doc/modules/model_evaluation.rst

```diff
@@ -1496,30 +1496,24 @@ Brier score loss
 
 The :func:`brier_score_loss` function computes the
 `Brier score <https://en.wikipedia.org/wiki/Brier_score>`_
-for binary classes. Quoting Wikipedia:
+for binary classes [Brier1950]_. Quoting Wikipedia:
 
     "The Brier score is a proper score function that measures the accuracy of
     probabilistic predictions. It is applicable to tasks in which predictions
     must assign probabilities to a set of mutually exclusive discrete outcomes."
 
-This function returns a score of the mean square difference between the actual
-outcome and the predicted probability of the possible outcome. The actual
-outcome has to be 1 or 0 (true or false), while the predicted probability of
-the actual outcome can be a value between 0 and 1 [Brier1950]_.
-
-The Brier score loss is also between 0 to 1 and the lower the score (the mean
-square difference is smaller), the more accurate the prediction is. It can be
-thought of as a measure of the "calibration" of a set of probabilistic
-predictions.
+This function returns the mean squared error of the actual outcome
+:math:`y \in \{0,1\}` and the predicted probability estimate
+:math:`p = \operatorname{Pr}(y = 1)` (``predict_proba``) as score:
```

Comment:
We like to link to the glossary when it might make sense

```suggestion
:math:`p = \operatorname{Pr}(y = 1)` (:term:`predict_proba`) as score:
```

I would also add "as outputed by ..."

Reply:
I replaced "as score" by "as outputted by" because, as you pointed out, it is a loss, not a score in scikit-learn terms.